mne/forward/_lead_dots.py

```diff
@@ -266,6 +229,7 @@ def _fast_sphere_dot_r0(r, rr1_orig, rr2s, lr1, lr2s, cosmags1, cosmags2s,
 
     # outer product, sum over coords
     ct = np.einsum('ik,jk->ij', rr1_orig, rr2)
+    np.clip(ct, -1, 1, ct)
```

Comment:
There was a value at something like `1.0000000000004` and SciPy is strict about their bounds